frontend/dockerfile/dockerfile2llb/convert.go

```diff
@@ -1066,6 +1068,9 @@ func dispatchCopy(d *dispatchState, cfg copyConfig) error {
 				a = a.Copy(st, f, dest, opts...)
 			}
 		} else {
+			if cfg.parents && !parentsEnabled {
+				return errors.New("flag --parents for ADD/COPY instructions requires the labs channel")
```

Comment:
I think we don't want an explicit failure message, I think we should aim to have a completely identical behavior as the previous version for the stable channel, instead of explicitly erroring the build.

Reply:
Fixed